src/styles/partials/components/_alt-list.scss

```diff
@@ -0,0 +1,56 @@
+ol.dg_alt-list {
+  list-style-type: none;
+  margin-bottom: $margin-large;
+  margin-left: 0;
+  padding-left: 0;
+  text-decoration: none;
+}
+
+ul.dg_alt-list {
+  list-style-type: none;
+  margin-bottom: $margin-large;
+  margin-left: 0;
+  padding-left: 0;
+  text-decoration: none;
+}
+
+.dg_i-float-right {
+  color: $gray-light;
+  float: right;
+  font-size: $large-icon-size;
+}
+
+.dg_alt-list {
+  li {
+    border-bottom: $border-width-small solid $gray-light;
+    margin-top: $default-margin;
+    -webkit-transition: all 0.3s ease 0s;
+    -moz-transition: all 0.3s ease 0s;
+    -o-transition: all 0.3s ease 0s;
+    text-decoration: none;
+    transition: all 0.3s ease 0s;
+    a {
+      color: $blue;
+      font-family: $default-heading-font;
+      font-size: $smallest-heading-font-size;
+      font-weight: $font-bold;
+      line-height: 30px;
+      text-decoration: none;
+      text-transform: uppercase;
+    }
+    &:hover {
```

Comment:
you prlbably want to include this block above the a. Reason being this directly applies to the li, where a is a child.

Reply:
Moved all of the block up to apply to the li, with the exception of the text-decoration property. The <a> tag needs to be targeted with the text-decoration tag directly in order to remove default link styles. Adding text-decoration to the parent, li, will not remove default link styling, such as underline. Will upload with a new commit shortly.